sources/cfsignal/README.md

```diff
@@ -0,0 +1,24 @@
+# cfsignal
+
+Current version: 0.0.1
+
+### Introduction
+
+Cfsignal is similar to [cfn-signal] (https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-signal.html).
+
+When creating an Auto Scaling Group, CloudFormation can be configured to wait for the expected number of signals from instances before considering the ASG successfully created. See [CreationPolicy](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-attribute-creationpolicy.html) and [UpdatePolicy](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-attribute-updatepolicy.html) for more details.
+
+Cfsignal uses `systemctl is-system-running` to determine whether the boot has succeeded or failed, and sends the corresponding signal to the CloudFormation stack.
+
+### configuration
+
+Configuration is read from a TOML file, which is generated from Bottlerocket settings:
+`should_send`: Whether to check system status and send signal.
+`stack_name`: Name of the CloudFormation Stack to signal.
+`logical_resource_id`: The logical ID of the AutoScalingGroup resource that you want to signal.
```

Comment:
These edits should go into the top of `cfsignal/src/main.rs` since that is where this README.md is generated from.
After triggering the `build.rs` script via a `cargo check` or build, this README file gets generated. So both places should have the same edits.